core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-state-controller-seo/dot-edit-page-state-controller-seo.component.ts

```diff
@@ -203,7 +206,7 @@ we want to show the lock off so the new user can steal the lock
     }
 
     private getStateModeOptions(pageState: DotPageRenderState): SelectItem[] {
-        const items = this.variant ? this.getModesBasedOnVariant() : ['edit', 'preview'];
+        const items = this.variant ? this.getModesBasedOnVariant() : ['edit', 'preview', 'live'];
```

Comment:
@hmoreras Same as before, we removed Live in the new Toolbar

Reply:
done